### ISSUE TITLE
Project JS Does Not like @steventhanna

### ISSUE THREAD

**Issue by steventhanna** [sotch-pr35mac/skrum#6]
So, I tried gathering the values from their respective places, and I don't know, but some shit is not working...

```js
$(document).ready(function() {
  $("#addProjectButton").click(function() {
    var name = $("#addProjectName").val();
    var description = $("#addProjectDescription").val();
    alert(name + " " + description);
    console.log(name);
    console.log(description);

    var postObj = {
      projectName: name,
      projectDescription: description
    };

    $.ajax({
      type: 'POST',
      url: '/project/create',
      data: postObj,
      success: function(data) {
        if (data.success == true) {
          console.log(data);
          // alert(data);
          window.location.reload();
        } else {
          alert("There was an error. Please try again.");
        }
      }
    });
  });
});
```
Here is my code.... I would like to know why it doesn't work, so I can do this busy work for you, @sotch-pr35mac in the future.

**Comment by sotch-pr35mac** [comment]
so what isn't working about it?

**Comment by sotch-pr35mac** [comment]
try something like this: 

```
$(document).ready(function() {
  $("#addProjectButton").click(function() {
    var name = $("#addProjectName").val();
    var description = $("#addProjectDescription").val();
    alert(name + " " + description); //we need to eventually change this to swal()
    console.log(name);
    console.log(description);
    var postObj = {
      projectName: name,
      projectDescription: description
    };
    $.ajax({
      type: 'POST',
      url: '/project/create',
      data: postObj,
      success: function(data) {
        if (data.success == true) {
          console.log(data);
          // alert(data); swal()
          window.location.reload();
        } else {
          alert("There was an error. Please try again."); //swall
        }
      },
     error: function(data) {
         swal("Uh-oh!", "There was an unexpected error.", "error");
         console.log(data);
     }
    });
  });
});
```

**Comment by sotch-pr35mac** [comment]
but im going to need more to go on why it isn't working or what isnt working about it if you want any more help with it

**Comment by sotch-pr35mac** [comment]
@steventhanna who wrote the code-snippet you posted originally?

**Comment by steventhanna** [comment]
I originally wrote the code snippet, however I based my code off of glenbardwest-athletics.  

In regards to what is actually happening: 
I have tried previously `console.log` and `alert` to try and see if information is actually being harvested from the page, but I could not get anything to work.  After clicking the button, the page would automatically refresh, like nothing happened.  I know the script is loaded onto the page, because I had an alert pop up once the page fully loaded. 

Whenever I tried sails development, this was always the part that stumped me.  I could never get the data from the page.
